content/api/templates.apib

```diff
@@ -4,179 +4,131 @@ description: Manage reusable content templates that are run through the SparkPos
-|headers| JSON | JSON dictionary containing headers other than `Subject`, `From`, `To`, and `Reply-To`  | no | See the [Header Notes](#header-header-notes). |
-
+Templates are a way to store and edit email content.
+Instead of passing inline content to a [Transmission](/api/transmissions) request, a template can be set to provide the content.
+All email content in a template (from, headers, text, and html) supports [substitution syntax](/api/substitutions-reference) for personalized emails.
+
+## Drafts & Published versions
+
+Templates can have both a draft and a published version. Transmissions send the published version by default.
+This allows you to work on an upcoming version of the content while continuing to send the last published version.
+Once ready, [publish the draft](#templates-publish-a-draft-put) to the start sending the new version without needing to change the transmission request.
+If needed, it's also possible to update the published version of a template directly, without affecting the draft version.
+
+## Template Object
+
++ Data Structure: Attributes
+    + id (string) - Unique alphanumeric ID.
+    + name (string) - Display name.
+    + description (string) - Description of the template.
+    + content (object) - Content that will be used to construct an email. Substitution syntax is supported in all content attributes.
+        + html (string) - HTML content for the email's `text/html` MIME part.
+        + text (string) - Text content for the email's `text/plain` MIME part.
+        + subject (string, required) - Email subject line.
+        + from (enum) - Email address used to compose the email's `From` header. Can be an object with `name` & `email` attributes: `{ "name": "My Company", "email": "deals@company.com" }`.
+            + (object)
+            + (string)
+        + reply_to (string) - Email address used to compose the email's `Reply-To` header.
+        + headers (object) - Object containing headers other than `Subject`, `From`, `To`, and `Reply-To`. See the [Header Notes](#header-header-notes).
+    + published (boolean) - Whether the template is published or is a draft version. A template cannot be changed from published to draft.
+    + options (object) - Object containing template options. All template options can be overwritten in a transmission's options.
+        + transactional (boolean) - Distinguish between transactional and non-transactional messages for unsubscribe and suppression purposes.
+        + open_tracking (boolean) - Enable or disable open tracking.
+        + click_tracking (boolean) - Enable or disable click tracking.
+    + has_draft (boolean) -  Whether the template has a draft version.
+    + has_published (boolean) -  Whether the template has a published version.
+    + last_update_time (string) - The last date any version of this template (draft of published) was updated.
```

Comment:
'of' typo. Should be 'or'.

Reply:
No, but I'll update it to be like the others you suggested